CIScripts/Report/Repair-NUnitReport.ps1

```diff
@@ -70,3 +108,20 @@ function Remove-Nodes {
         $_.ParentNode.RemoveChild($_)
     } | Out-Null
 }
+
+function Get-NameOfPesterTestSuite {
+    Param([Parameter(Mandatory = $true)] [System.Xml.XmlElement] $FromNode)
+    $PesterFilepath = $KeptNode."name"
+    # Example:
+    # From "C:/SomePath/TestSuiteName.Tests.ps1"
+    # we get "TestSuiteName"
+    $SuiteName = (Split-Path $PesterFilepath -Leaf).split('.')[-3]
+    return $SuiteName
+}
+
+function Copy-NodeOfSpecificTestSuite {
+    Param([Parameter(Mandatory = $true)] [XML] $FromNode)
+    $PesterFilesNodes = $RootOfClonedXML.Node.FirstChild.ChildNodes
```

Comment:
Shouldn't it be `$FromNode`?

Reply:
But `$FromNode` should be used in place of `$RootOfClonedXML` if we undertand the code correctly.